R/run_on.R

```diff
@@ -0,0 +1,10 @@
+run_on <- function(linebreak = FALSE) {
```

Comment:
Suggestion for documentation. Using `@noRd` is the tidyverse style guide recommendation - https://style.tidyverse.org/documentation.html#internal-functions.

You will also need to run `devtools::document()` manually since the actions aren't working because of {captioner}.
```suggestion
#' What system is the code being run on
#'
#' Collates RStudio and R version information into
#' a character vector which can be used elsewhere.
#'
#' @param linebreak (Default: FALSE) - Should a 
#' linebreak be appended to the 'run on' text?
#'
#' @noRd
run_on <- function(linebreak = FALSE) {
```